packages/dashboard-core-plugins/src/panels/MarkdownPanel.tsx

```diff
@@ -17,8 +23,12 @@ import { RootState } from '@deephaven/redux';
 import Panel from './Panel';
 import MarkdownContainer from '../controls/markdown/MarkdownContainer';
 import MarkdownStartPage from '../controls/markdown/MarkdownStartPage';
-import MarkdownEditor from '../controls/markdown/MarkdownEditor';
 import './MarkdownPanel.scss';
+import type MarkdownEditorType from '../controls/markdown/MarkdownEditor';
+
+const MarkdownEditor = lazy(
+  () => import('../controls/markdown/MarkdownEditor')
+);
```

Comment:
You'll need to wrap the usage of `MarkdownEditor` with a `Suspense` in the `render` method so it has an element to fallback to, e.g.
```
            <Suspense fallback={<LoadingOverlay />}>
              <MarkdownEditor
                ...
              />
            </Suspense>
```
That will show a loading spinner while it's loading, and fix the unit test that is broken. Similar thing will be needed in `NotebookPanel`.

https://react.dev/reference/react/lazy#usage